lib/uniffi-zcash/src/udl/zcash_primitives/legacy/keys/account_pub_key.udl

```diff
@@ -1,3 +1,25 @@
 interface ZcashAccountPubKey {
-  // todo
+
+  [Throws=ZcashError]
+  constructor(sequence<u8> data);
```

Comment:
This previously was "deserialize" method in the original object. Modified for accomplishing with our [code conventions](https://github.com/eigerco/uniffi-zcash-lib/wiki/Bindings-source-code-conventions).